src/org/daisy/dotify/translator/impl/sv_SE/sv_SE-single-character-contracted-words.xml

```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file contains more than single character contracted brailles, but will be updated.